src/packetizer.iced

```diff
@@ -53,22 +53,27 @@ exports.Packetizer = class Packetizer
   ERR : -1
 
   ##-----------------------------------------
-  
+
   constructor : ->
     @_ring = new Ring()
     @_state = @FRAME
     @_next_msg_len = 0
 
   ##-----------------------------------------
-  
+
   send : (msg) ->
     b2 = pack msg
     b1 = pack b2.length
     bufs = [ b1, b2 ]
     rc = 0
     enc = 'binary'
-    for b in bufs
-      @_raw_write b.toString(enc), enc
+
+    if @_raw_write_bufs
```

Comment:
I did this just to get things work, we could likely move it back but we'd have to change the electron/rn code